ui/src/app/views/cohort-list/component.spec.ts

```diff
@@ -103,6 +105,7 @@ describe('CohortListComponent', () => {
       ] }).compileComponents().then(() => {
         cohortListPage = new CohortListPage(TestBed);
       });
+    registerApiClient(CohortsApi, new CohortsApiStub());
```

Comment:
What I have found is that you will need to stub out a mock response for any api calls your component makes. If that isn't in place, then the stub will throw an error. See what I did for the profile api here: https://github.com/all-of-us/workbench/pull/1710/files#diff-8a1055dcdba0f29945195b2d731972f8

Reply:
I figured it out, thanks!!